readme.md

```diff
@@ -568,6 +568,7 @@
 - [EOSIO](https://github.com/DanailMinchev/awesome-eosio#readme) - A decentralized operating system supporting industrial-scale apps.
 - [Corda](https://github.com/chainstack/awesome-corda#readme) - Open source blockchain platform designed for business.
 - [Waves](https://github.com/msmolyakov/awesome-waves#readme) - Open source blockchain platform and development toolset for Web 3.0 apps and decentralized solutions.
+- [Substrate](https://github.com/substrate-developer-hub/awesome-substrate#readme) - Framework for writing scalable, upgradeable blockchains in Rust; crafted with ❤️ by [Parity Technologies](https://github.com/paritytech).
```

Comment:
I believe the part after the `;` (crafted with ❤️ by [Parity Technologies](https://github.com/paritytech)) is noise.

Reply:
You have not run `awesome-lint` on the list.

Reply:
Thank you for pointing out the the `awesome-lint` problem :pray: Personally I do not believe the link to Parity's GitHub organization is noisy because it adds context to the entry and the two links do not refer to the same GitHub organization. I'm happy to remove it if other people agree that it is noisy.

Reply:
@danforbes It says on your GitHub profile that you are a *developer advocate* for Parity Technologies. Are you not in conflict of interest?